Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Two-handed Generic Weapons

#### Changelog:
- Added 5 Spears
- Added 5 Halberds

#### How to test:
- Check that random pictures and names are being set properly
- You can search character with specific artifact by typing artifact name in character search (e.g. `spear` or `halberd`)
